geoips/plugins/modules/procflows/config_based.py

```diff
@@ -1691,10 +1690,12 @@ def call(fnames, command_line_args=None):
     failed_compares = {}
     for cpath in final_products:
         if cpath != "no_comparison":
-            curr_compare_outputs = find_entry_point(
-                "output_comparisons", final_products[cpath]["compare_outputs_module"]
+            from geoips.interfaces.module_based.output_checkers import output_checkers
+
+            output_checker = output_checkers.get_checker(
+                final_products[cpath]["files"][0]
             )
-            curr_retval = curr_compare_outputs(cpath, final_products[cpath]["files"])
+            curr_retval = output_checker.call(cpath, final_products[cpath]["files"])
```

Comment:
Maybe this should be "select_checker" or "identify_checker", to differentiate from the standard "get" interface method

Ie, like all the other interfaces, if you do "output_checkers.get("geotiff_checker") it will return the explicitly named geotiff_checker.  We can use that if we want to eventually specify which checker specifically we should use.

We can attach a helper function called 
``checker_plugin_name = output_checkers.identify_checker("<filename>")``
, which will return, say "geotiff_checker", then you can do
``checker_plugin.get(checker_plugin_name)``

Reply:
That's not a bad idea. Or we can create just one function like "select_checker" as you mentioned above, which automatically gets the checker for us. Seems unnecessary to have it split up into two functions, however I'm happy to implement whichever you think is more appropriate. If we end up wanting to grab a checker via get, the two method approach might be better suited.

Reply:
Either way is fine. output_checkers.get() is already a built in interface function that is on all interfaces, right @jsolbrig ?